sql/dd/impl/sdi.cc

```diff
@@ -375,37 +375,38 @@ Sdi_type serialize(const Tablespace &tablespace) {
    compatibility check: MYSQL_VERSION less than or equal, dd_version
    equal, and sdi_version equal.
 */
-bool CheckDefaultCompatibility(const RJ_Document &doc) {
+bool CheckDefaultCompatibility(const RJ_Document &doc __attribute__((unused))) {
   assert(doc.HasMember("mysqld_version_id"));
```

Comment:
warning: unused variable 'mysqld_version_id' [-Wunused-variable]
but this assert is using it? :thinking:

Reply:
yes, I think the assertion goes to a macro and the compiler does not recognize it

Reply:
I see